.eslintrc.js

```diff
@@ -1,8 +1,15 @@
 module.exports = {
     'env': {
         'browser': true,
-        'jquery': true
+        'es6': true,
+        "jest/globals": true,
+        'jquery': true,
+        "node": true
     },
+    'parserOptions': {
+        'ecmaVersion': 6
+    },
+    "plugins": ["jest"],
```

Comment:
> Or does the plugin enable some of its lint rules by default? Or maybe this is just so that you can have the jest/globals in env above.

It adds some of its own rules, but also adds the relevant globals so eslint does not complain about undefined globals such as `describe` and `it`